harold/plugins/http.py

```diff
@@ -46,14 +46,15 @@ def _authenticate_request(self, request):
             raise AuthenticationError
 
     def render_GET(self, request):
+        response = ""
         try:
             self._authenticate_request(request)
         except AuthenticationError:
             request.setResponseCode(403)
         else:
-            self._handle_request(request)
+            response = self._handle_request(request)
```

Comment:
Need to make sure we're sure to return the `NOT_DONE_YET` response:
https://github.com/spladug/harold/pull/42/files#diff-270925a10931db85f0b8727a98cef1edR232